lib/common/formats.js

```diff
@@ -902,7 +903,7 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
   },
 
   /**
-   * Creates a Swift implementation file of a class with values
+   * Creates a Swift implementation file of a class with values. It adds default `class` object type, `public` access control and `UIKit` import.
    *
    * @memberof Formats
    * @kind member
```

Comment:
Can we add documentation here for the new file options?

Reply:
i added doc in the param, do you think i missed something?

Reply:
Nope, looks good!